test/screens/v2/screen_data/parameters_test.exs

```diff
@@ -64,12 +65,19 @@ defmodule Screens.V2.ScreenData.ParametersTest do
   describe "audio_interval_minutes/1" do
     test "is nil for a screen type without periodic audio" do
       static_params = build_params(periodic_audio: nil)
-      assert Parameters.audio_interval_minutes(@app_id, static_params) == nil
+      screen = build_screen()
+      assert Parameters.audio_interval_minutes(screen, static_params) == nil
     end
 
     test "is the configured interval for a screen type with periodic audio" do
       static_params = build_params(periodic_audio: build_periodic_audio(interval_minutes: 7))
-      assert Parameters.audio_interval_minutes(@app_id, static_params) == 7
+      screen = build_screen()
+      assert Parameters.audio_interval_minutes(screen, static_params) == 7
+    end
+
+    test "is 0 for a screen type with audio interval disabled" do
```

Comment:
I think this description should be "for a _screen_ with audio interval disabled", since it's the individual screens that have this flag, not the whole screen type.

Reply:
good point, agreed